### ISSUE TITLE
Crashes when searching for google image on Windows 10

### ISSUE THREAD

**Issue by adamelmsri** [realmayus/AutoSkribbler#2]
Works with local images.

Always ends with this exit code when trying to select image from google.
**Process finished with exit code -1073740791 (0xC0000409)**

**Comment by realmayus** [comment]
Do you have the "use image workaround" checkbox ticked?

**Comment by adamelmsri** [comment]
I don't think so,
Where can i find it?

**Comment by realmayus** [comment]
It should appear in the main window above the buttons

**Comment by adamelmsri** [comment]
![Capture24](https://user-images.githubusercontent.com/54114251/82212920-60324100-9924-11ea-8cb4-01f78386a1dc.PNG)

![Capture25](https://user-images.githubusercontent.com/54114251/82213666-9cb26c80-9925-11ea-9b0d-07b4ae88a4d6.PNG)


This is all i see, could be a compatibility problem as you are using linux and I am on windows.

**Comment by realmayus** [comment]
hmm, this is weird, the checkbox _should_ appear. I'll see what I can do.

**Comment by realmayus** [comment]
Ok, my bad, forgot to push my changes to GitHub :P
Could you please pull the changes and check if it works with the checkbox ticked/unticked?

**Comment by adamelmsri** [comment]
I now have the "use image workaround" button, but it still exits with the same code.

**Process finished with exit code -1073740791 (0xC0000409)**

Do you have a windows virtual machine to test it yourself ?

**Comment by ar06x** [comment]
~~after a bit of debugging i managed to get this `selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home` i have no idea what the PATH variable needs to be in windows~~

Edit: Turns out I just need to change this `WEBDRIVER_PATH = "/home/marius/chromedriver"`
I do get a `Segmentation fault` when I click the image I want to draw, seems that only local images work...

**Comment by ar06x** [comment]
```python
    def on_img_select(self, event, pil_img_obj: Image):
        # print(type(pil_img_obj))
        # pil_img_obj.show()
        print("Selected image!")
        self.main_window.imgObj = pil_img_obj
        self.main_window.preferLocalImg = False
        self.main_window.reload_img_preview()
        self.close()
```
the `self.close()` causes that to happen apparently

**Comment by realmayus** [comment]
Thanks, I just changed the webdriver path to the current directory. I'll look into the issue with the segmentation fault 👍

**Comment by ar06x** [comment]
Yeah, I dont know much python, but I tried to fix it with no success. Was
able to get it to work with loading 1 image. But after drawing that image
and trying to load 1 other image, it would crash.
